src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1409,6 +1409,16 @@ void NetworkOPsImp::switchLastClosedLedger (
     clearNeedNetworkLedger ();
     newLedger->setClosed ();
     auto openLedger = std::make_shared<Ledger> (false, std::ref (*newLedger));
+    // Apply transactions from the current open ledger to
+    // this new one, including locally held transactions.
+    {
+        // VFALCO Which hash do we use?
```

Comment:
Oh!  So you're actually saying that this code isn't right, because you have the wrong hash?!